src/blog/2024/01/flowfuse-release-2-0.md

```diff
@@ -0,0 +1,54 @@
+---
+title: FlowFuse 2.0 Release
+subtitle: Elevating Node-RED Device Management to new heights
+description: Announcing the launch of FlowFuse 2.0, a significant upgrade in managing Node-RED remote instances.
+date: 2024-01-18
+authors: ["marian-demme"]
+image: /blog/2024/01/images/flowfuse-2-release.png
+tags:
+    - posts
+    - flowfuse
+    - news
+    - releases
+---
+
+Following the release of FlowFuse 1.0 end of 2022, we're excited to release FlowFuse 2.0, marking a significant step in managing Node-RED remote instances, which we call Devices. 
+<!--more-->
+
+Many organizations position Node-RED instances on remote servers like edge or industrial devices. This way they can meet network requirement, interact with analog protocols, and overcome other infrastructure requirements. Management of remote instances is crucial for the overall success of closing the gap between IT and OT. A key enhancement was the introduction of Device Groups (from version 1.15) and the new feature to assign target snapshots. This allows for direct and streamlined management of Node-RED Device fleets, setting the stage for future advancements in device management capabilities.
+
+For our FlowFuse users, this means it is no longer necessary or recommended to assign devices to an instance. Node-RED devices can be managed independently, and snapshots can be assigned via DevOps pipelines.
+
+## Enterprise-Readiness
+
+FlowFuse is committed to augmenting the enterprise-readiness of Node-RED with introductions like [Single Sign-On (SSO)](https://flowfuse.com/docs/admin/sso/), [Multi-Factor Authentication (MFA)](https://flowfuse.com/docs/user/user-settings/#two-factor-authentication), and [High Availability](https://flowfuse.com/docs/user/high-availability/) since version 1.0. Furthermore, we recently achieved [SOC2 Type 1 compliance](https://flowfuse.com/blog/2024/01/soc2/). With these advancements, Node-RED, in combination with FlowFuse, is genuinely ready for enterprise and production use.
+
+## Enhanced Integration Capabilities
+The Node-RED Flow Library has always been a cornerstone, offering over 4800 connectors (nodes) for various OT and IT protocols. Thanks to the community and the Node-RED library. Building on this foundation, FlowFuse introduced "Certified Nodes" and "Blueprints". These [Blueprints](https://flowfuse.com/blog/2023/10/blueprints/) are designed to provide an easier start with Node-RED, showcasing its full potential, while Certified Nodes ensure the security of the nodes used. Learn more about our new Certified Nodes [here](https://flowfuse.com/blog/2023/10/certified-nodes/).
+
```

Comment:
```suggestion

## Developer Velocity

FlowFuse allowed teams to have access to the same Node-REDs, and collaborate with development. To speed up teams we've worked hard on maturing our [snapshot capabilties](todo) and build a full [DevOps Pipeline](todo). This provides team the option of developing their flows in a development instance, and have a QA review before any changes make it to production.

<todo chat about more dev velocity features>
```

Reply:
How about @ZJvandeWeg :
```suggestion
## Developer Velocity

Node-RED team development is made possible with FlowFuse. Different development team members are able to share and collaborate on the same Node-RED instance. This makes for much easier collaboration between Node-RED developers.  We've worked hard on maturing our [snapshot capabilties](https://flowfuse.com/docs/user/snapshots/) and introduced  [DevOps Pipelines](https://flowfuse.com/docs/user/devops-pipelines/) that can be set up to stage Node-RED instances that have different development stages, e.g. test, development and production.
```

Reply:
Double space before `[DevOps]`, otherwise it's great!